data/darktableconfig.xml.in

```diff
@@ -1009,8 +1009,8 @@
       </enum>
     </type>
     <default>show both</default>
-    <shortdescription>show search module text entry</shortdescription>
-    <longdescription/>
+    <shortdescription>show module groups and/or search text entry</shortdescription>
+    <longdescription>show module groups and/or search text entry. (needs a restart)</longdescription>
```

Comment:
if it really needs restart, shouldn't you also change to
```xml
<dtconfig prefs="darkroom" restart="true">
```
couple lines above.

Reply:
Thanks for the tip. Made the change.